src/extension.ts

```diff
@@ -203,7 +203,7 @@ class ExtensionManager implements vscode.Disposable {
       context.ignoreCMakeListsMissing = !fullFeatureSet;
       this._statusBar.setVisible(fullFeatureSet);
     } else {
-      log.info(localize('enableFullFeatureSet.no.active.folder', 'enableFullFeatureSet(\'{fullFeatureSet}\') called but not active folder found.'));
+      log.error(localize('enableFullFeatureSet.no.active.folder', 'enableFullFeatureSet({0}) called but not active folder found.'), fullFeatureSet);
```

Comment:
Sorry, I said `error` at first, but then realized it should be `debug` or `trace`

Reply:
Right.